packages/p2p/src/pages/orders/chat/chat-footer/chat-footer-icon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+
+type TChatFooterIcon = {
+    should_show_attachment_icon: boolean;
```

Comment:
should this be optional instead? Since from line 10 and 11 it looks like `should_show_attachment_icon` could be undefined
```suggestion
    should_show_attachment_icon?: boolean;
```